### PR TITLE
AST-1047 - Product Variation - orientation option

### PR DESCRIPTION
**Pull request by GauravKhupse**
### Screenshots
Referance:- https://share.bsf.io/E0uKEWD7
Added:- https://share.bsf.io/Qwu48jOn

### Types of changes

 New feature (non-breaking change which adds functionality) 

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/compatibility/woocommerce/class-astra-woocommerce.php [hunk 1]
```diff
@@ -1689,11 +1692,35 @@ public function add_styles() {
...
+						'padding-top'    => '8px',
+					),
+				);
+
+				$css_output .= astra_parse_css( $css_output_woo_variation_layout );
+			}
+
+
+
+
```
Comment (imnavanath) [inline inc/compatibility/woocommerce/class-astra-woocommerce.php:1723]:
```suggestion
```

File: inc/compatibility/woocommerce/class-astra-woocommerce.php [hunk 2]
```diff
@@ -1689,11 +1692,35 @@ public function add_styles() {
...
+			$woo_variation_layout = astra_get_option( 'single-product-variation-tabs-layout' );
+
+			if ( 'horizontal' === $woo_variation_layout ) {
+				$css_output_woo_variation_layout = array(
+					'.woocommerce div.product form.cart .variations td' => array(
+						'display'     => 'table-cell',
+						'padding-top' => '8px',
+					),
+					'.woocommerce div.product form.cart .variations td.label' => array(
```
Comment (imnavanath) [inline inc/compatibility/woocommerce/class-astra-woocommerce.php:1712]: can we minimize the selector weightage here?

File: inc/compatibility/woocommerce/class-astra-woocommerce.php [hunk 3]
```diff
@@ -1689,11 +1692,35 @@ public function add_styles() {
...
 			
+			/**
+			 * Single page variation tab layout.
+			 */
+
+			$woo_variation_layout = astra_get_option( 'single-product-variation-tabs-layout' );
+
+			if ( 'horizontal' === $woo_variation_layout ) {
+				$css_output_woo_variation_layout = array(
+					'.woocommerce div.product form.cart .variations td' => array(
```
Comment (imnavanath) [inline inc/compatibility/woocommerce/class-astra-woocommerce.php:1708]: can we minimize the selector weightage here?

File: inc/compatibility/woocommerce/class-astra-woocommerce.php (hunk 1 above)
Comment (imnavanath) [reply inc/compatibility/woocommerce/class-astra-woocommerce.php:1723]: Remove the extra blank lines added here.

Comment (GauravKhupse) [reply inc/compatibility/woocommerce/class-astra-woocommerce.php]: Changes done!

File: inc/compatibility/woocommerce/class-astra-woocommerce.php (hunk 2 above)
Comment (GauravKhupse) [reply inc/compatibility/woocommerce/class-astra-woocommerce.php:1712]: Actually, we have overridden the Woocommererce property for the same selector.

File: inc/compatibility/woocommerce/class-astra-woocommerce.php (hunk 3 above)
Comment (GauravKhupse) [reply inc/compatibility/woocommerce/class-astra-woocommerce.php:1708]: Actually, we have overridden the Woocommererce property for the same selector.

**Review by imnavanath — APPROVED** [review]

File: inc/compatibility/woocommerce/class-astra-woocommerce.php [hunk 4]
```diff
@@ -453,6 +453,9 @@ public function theme_defaults( $defaults ) {
 			$defaults['shop-archive-width']     = 'default';
 			$defaults['shop-archive-max-width'] = 1200;
 
+
```
Comment (premanshup) [inline inc/compatibility/woocommerce/class-astra-woocommerce.php:456]:
```suggestion
```

File: inc/compatibility/woocommerce/class-astra-woocommerce.php [hunk 5]
```diff
@@ -453,6 +453,9 @@ public function theme_defaults( $defaults ) {
 			$defaults['shop-archive-width']     = 'default';
 			$defaults['shop-archive-max-width'] = 1200;
 
+			
+			$defaults['single-product-variation-tabs-layout'] = 'vertical';
+
```
Comment (premanshup) [inline inc/compatibility/woocommerce/class-astra-woocommerce.php:458]:
```suggestion
```

File: inc/compatibility/woocommerce/class-astra-woocommerce.php (hunk 4 above)
Comment (GauravKhupse) [reply inc/compatibility/woocommerce/class-astra-woocommerce.php]: Changes done!

File: inc/compatibility/woocommerce/class-astra-woocommerce.php (hunk 5 above)
Comment (GauravKhupse) [reply inc/compatibility/woocommerce/class-astra-woocommerce.php:457]: Chnages done!

**Pull request merged**